tokio/src/signal/windows/sys.rs

```diff
@@ -25,24 +37,33 @@ fn new(signum: DWORD) -> io::Result<RxFuture> {
 
 #[derive(Debug)]
 pub(crate) struct OsStorage {
-    ctrl_c: EventInfo,
     ctrl_break: EventInfo,
+    ctrl_close: EventInfo,
+    ctrl_c: EventInfo,
+    ctrl_logoff: EventInfo,
+    ctrl_shutdown: EventInfo,
 }
 
 impl Init for OsStorage {
     fn init() -> Self {
         Self {
-            ctrl_c: EventInfo::default(),
-            ctrl_break: EventInfo::default(),
+            ctrl_break: Default::default(),
+            ctrl_close: Default::default(),
+            ctrl_c: Default::default(),
+            ctrl_logoff: Default::default(),
+            ctrl_shutdown: Default::default(),
         }
     }
 }
 
 impl Storage for OsStorage {
     fn event_info(&self, id: EventId) -> Option<&EventInfo> {
         match DWORD::try_from(id) {
-            Ok(CTRL_C_EVENT) => Some(&self.ctrl_c),
             Ok(CTRL_BREAK_EVENT) => Some(&self.ctrl_break),
+            Ok(CTRL_CLOSE_EVENT) => Some(&self.ctrl_close),
+            Ok(CTRL_C_EVENT) => Some(&self.ctrl_c),
+            Ok(CTRL_LOGOFF_EVENT) => Some(&self.ctrl_logoff),
+            Ok(CTRL_SHUTDOWN_EVENT) => Some(&self.ctrl_shutdown),
```

Comment:
These need to be qualified now

```suggestion
            Ok(wincon::CTRL_BREAK_EVENT) => Some(&self.ctrl_break),
            Ok(wincon::CTRL_CLOSE_EVENT) => Some(&self.ctrl_close),
            Ok(wincon::CTRL_C_EVENT) => Some(&self.ctrl_c),
            Ok(wincon::CTRL_LOGOFF_EVENT) => Some(&self.ctrl_logoff),
            Ok(wincon::CTRL_SHUTDOWN_EVENT) => Some(&self.ctrl_shutdown),
```

Reply:
Sorry, I'm developing on a Mac and have been using the tokio CI as my cargo check :) Just got a windows machine to test out the new signals.